config/settings/base.py

```diff
@@ -281,6 +276,8 @@
 CORS_URLS_REGEX = r"^/api/.*$"
 # Your stuff...
 # ------------------------------------------------------------------------------
+# Are you using local passwords or oidc?
```

Comment:
I've been realizing that we need to do a better job of documenting the environment variable options. To keep the problem from getting worse for the time being, can you add something to the docs that specifies that this must either be "local" or "oidc" (and that it defaults to local)?

Reply:
Added some basic documentation for now. We have current issues open to do a better job of centralizing and updating docs so I think this will do for now until we decided what we want this all to look like.